reducers/heroes.js

```diff
@@ -0,0 +1,18 @@
+import { heroes } from 'dotaconstants';
+import { API_HOST } from './../yasp.config';
+
+// lazy evaluation
+// add API_HOST to hero img path so components
+// don't need to know API_HOST
+const transformHeroImagePath = (heroes) => {
+  const heroList = {};
+  Object.keys(heroes).forEach((key) => {
+    heroList[key] = { ...heroes[key] };
+    heroList[key].img = API_HOST + heroList[key].img;
+  });
+
```

Comment:
As my current understanding selector used as to compute derivative data from state,
the reducer created because HeroesIndex needs list of all heroes, Rankings and Benchmarks too. in previous implementation I put this code to `connect'ed component HeroesIndex direct from dotaconstants, but this code make components harder to read. Other consideration for sake of good encapsulation, I don't want components to know where yasp data came from, so I put all data to state, and let component being presentational component as minimum as possible without knowing the source of data.